statsmodels/tsa/statespace/tests/test_varmax.py

```diff
@@ -532,9 +540,10 @@ def test_summary(self):
 
 
 class CheckFREDManufacturing(CheckVARMAX):
-    def __init__(self, true, order, trend, error_cov_type, cov_type='oim',
+    @classmethod
+    def setup_class(cls, true, order, trend, error_cov_type, cov_type='oim',
                  **kwargs):
-        self.true = true
+        cls.true = true
         # 1960:Q1 - 1982:Q4
         dta = webuse('manufac', 'http://www.stata-press.com/data/r12/')
```

Comment:
separate issue, not related to this PR

We don't always have internet access in a test run, e.g. Ubuntu doesn't allow it and errors here.
This will have to be replaced with locally storing the csv or dta file.